packages/core/src/browser/style/index.css

```diff
@@ -130,17 +130,14 @@ button.theia-button, .theia-button {
     min-width: 65px;
     outline: none;
     cursor: pointer;
-    padding-left: 9px;
-    padding-right: 9px;
-    padding-top: 4px;
-    padding-bottom: 4px;
+    padding: 4px 9px;
     margin-left: calc(var(--theia-ui-padding)*2);
     border-radius: 1px;
     box-shadow: 0px 1px 5px 0px rgba(0, 0, 0, 0.2), 0px 1px 2px 0px rgba(0, 0, 0, 0.14), 0px 1px 1px -2px rgba(0, 0, 0, 0.12);
     transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
 }
 
-button.theia-button:hover, .theia-button:hover {
+button.theia-button:hover, .theia-button:hover, .theia-button:focus {
```

Comment:
@jbicker have you checked how it affects other buttons when they are focused? is it fine?

Reply:
Yes. It looks fine to me. I like it.

Reply:
ok, just checking, thank you!